css/qmwp-connectors.css

```diff
@@ -1,234 +1,278 @@
-	background-color: black;
-	color: white;
-	height: 50px;
-	padding: 10px;
-	font-size: 20px;
+    background-color: black;
+    color: white;
+    height: 50px;
+    padding: 10px;
+    font-size: 20px;
 }
 
 .ui-dialog {
-	left: 0;
-	outline: 0 none;
-	padding: 0em !important;
-        z-index: 1000 !important;
+    left: 0;
+    outline: 0 none;
+    padding: 0em !important;
+    z-index: 1000 !important;
 }
 
 .imgClass {
-	background-color: transparent; background-image :
-	url("https://i.imgur.com/rbUDVrJ.png");
-	background-position: 0px 0px;
-	background-repeat: no-repeat;
-	background-size: 38px auto;
-	height: 37px;
-	margin-right: 35px;
-	margin-top: -5px;
-	padding: 6px 21px;
-	width: 23px;
-	background-image: url("https://i.imgur.com/rbUDVrJ.png");
+    background-color: transparent;
+    background-image: url("https://i.imgur.com/rbUDVrJ.png");
+    background-position: 0px 0px;
+    background-repeat: no-repeat;
+    background-size: 38px auto;
+    height: 37px;
+    margin-right: 35px;
+    margin-top: -5px;
+    padding: 6px 21px;
+    width: 23px;
+    background-image: url("https://i.imgur.com/rbUDVrJ.png");
 }
 
 .connectorInfoTable {
-	border-bottom: 2px solid;
-	margin-top: 20px;
-	width: 90%;
-	font-size: 13px;
+    border-bottom: 2px solid;
+    margin-top: 20px;
+    width: 90%;
+    font-size: 13px;
 }
 
 .tabletr {
-	border-top: 2px solid;
+    border-top: 2px solid;
 }
 
-.bold{
-	font-weight: bold;
+.bold {
+    font-weight: bold;
 }
 
 /*****************/
 /* updates table */
 /*****************/
 .updates-table tbody tr td {
-	vertical-align: middle;
+    vertical-align: middle;
 }
+
 .updates-table tbody tr td:first-child {
-	text-align: center;
+    text-align: center;
 }
+
 .updates-table tbody tr td img {
-	height: 25px;
+    height: 25px;
 }
+
 .updates-table tr td:last-child {
-	text-align: right;
+    text-align: right;
 }
+
 .updates-table thead tr {
-	border-bottom: 2px solid #29BDCA;
-	color: #29BDCA;
-	font-weight: bold;
+    border-bottom: 2px solid #29BDCA;
+    color: #29BDCA;
+    font-weight: bold;
 }
+
 .updates-table tbody tr:nth-child(even) {
-	background: #f0f0f0;
+    background: #f0f0f0;
 }
 
-
 /******************/
 /* get it here url */
 /******************/
 .getitnow-container {
-	border-top: 1px dashed black;
-	text-align: center;
-	padding-top: 5px;
-	margin-top: 10px;
+    border-top: 1px dashed black;
+    text-align: center;
+    padding-top: 5px;
+    margin-top: 10px;
 }
+
 .getitnow-container a {
-	font-weight: bold;
+    font-weight: bold;
+}
+
+#content .my-location {
+    padding: 15px 5px;
+    background-color: #fff;
+    margin: 0 auto;
+}
+
+/* Responsive connectors shortcode*/
+
+.qm-account-block-left {
+    width: 25%;
+    max-width: 132px;
+}
+
+.qm-account-block-right {
+    width: 70%;
+    float: left;
+    margin: 0 0 0 7px;
+}
+
+.qm-account-last-updated {
+    position: relative;
+    top: 0;
+    right: 0;
+}
+
+img.qm-connect-image {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Element (img.qm-connect-image) is overqualified, just use .qm-connect-image without element name.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1195596520/issues/source?bid=2836188&fileBranchId=2925382#l267)